Source/DafnyLanguageServer.Test/Synchronization/DiagnosticsTest.cs

```diff
@@ -582,7 +582,7 @@ modifies this
       Assert.AreEqual("This is the postcondition that might not hold.", relatedInformation[0].Message);
       Assert.AreEqual(new Range((14, 16), (14, 21)), relatedInformation[0].Location.Range);
       Assert.AreEqual("Related location", relatedInformation[1].Message);
-      Assert.AreEqual(new Range((9, 13), (9, 14)), relatedInformation[1].Location.Range);
+      Assert.AreEqual(new Range((9, 11), (9, 16)), relatedInformation[1].Location.Range);
```

Comment:
Could we have tests with more nested expressions as well?

Reply:
tests added